dashboard/client/index.js

```diff
@@ -41,8 +41,18 @@ Template.dashboard.onCreated(function () {
     }
   }
 
-  instance.getAggrData = opts => new Promise((resolve, reject) => {
-    Meteor.call('getAggr', opts, (err, res) => {
+  instance.getChartData = opts => new Promise((resolve, reject) => {
+    Meteor.call('getAggregatedData', opts, (err, res) => {
+      if (err) reject(err)
+      Meteor.call('parseDataForNvd', res, (err1, res1) => {
```

Comment:
This is a server-side method created to decrease the load on client-site (browser).

> That means, in this scenario, we are getting aggregated data from the server and then sending it back to the server to be parsed and returned again.

Right.